<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QDialog" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>WAITING!!!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>hello</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>560</y>
     <width>187</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>116</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortLineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>2333</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>187</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>set</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>671</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send_all_pushButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>560</y>
     <width>187</width>
     <height>57</height>
    </rect>
   </property>
   <property name="text">
    <string>send all</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
